zebedee-cms/pom.xml

```diff
@@ -183,6 +183,12 @@
             <version>0.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.onsdigital</groupId>
+            <artifactId>dp-jwt-verifier-java</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
This should be pinned to a specific version - otherwise it could change each time Zebedee is built which we don't want.
Update this to `v0.1.1` or whatever the latest release of this library is.

Reply:
Unable to action due to authorisation to JitPack 
this will need to be updated when the authorisation issues have been resolved

Reply:
This has now been resolved 
Thanks to MikeK